src/main/java/io/iron/ironworker/client/Client.java

```diff
@@ -123,9 +124,17 @@ public TaskEntity createTask(String codeName, Map<String, Object> params, Map<St
         if (params == null) {
             params = new HashMap<String, Object>();
         }
+        String payload = gson.toJson(params);
```

Comment:
you could extract duplicated code to new method `private String loadPayload(params, options)`